packages/docs/components/page-configs/va-button-dropdown/api-options.ts

```diff
@@ -0,0 +1,27 @@
+import { ManualApiOptions } from 'vuestic-ui/src/services/api-docs/ManualApiOptions'
```

Comment:
Without icon still shows icon on click. Is that correct behaviour?
![image](https://user-images.githubusercontent.com/5394573/102550372-cbb4da80-40ce-11eb-98ad-47b3b5c91590.png)
CLICK
![image](https://user-images.githubusercontent.com/5394573/102550355-c192dc00-40ce-11eb-82d3-08fa1c8c4532.png)